cms/templates/sudo/sudo.html

```diff
@@ -1,7 +1,8 @@
 {% block body %}
```

Comment:
Honestly, I don't recall at this point whether we discussed that. Do you remember why that wasn't going to be possible?